python/tests/test_tables.py

```diff
@@ -1401,6 +1401,21 @@ def test_metadata_schema(self, table_5row):
                 table_5row.assert_equals(table2)
             table_5row.assert_equals(table2, ignore_metadata=True)
 
+    def test_degenerate_metadata_schema(self, table_5row):
+        if hasattr(table_5row, "metadata_schema"):
+            table_5row.clear()
```

Comment:
Not sure we should be mutating the global fixture here? Better to copy first then clear?

Also, why are we checking if the fixture has a metadata_schema?